web/src/test/java/com/graphhopper/http/resources/RouteResourceCustomModelTest.java

```diff
@@ -225,6 +229,17 @@ public void customBikeShouldBeLikeJsonBike() {
         assertEquals(path.get("distance").asDouble(), 660, 10);
     }
 
+    @Test
+    public void testSubnetworkRemovalPerProfile() {
```

Comment:
Can we maybe add a few more tests where this PR makes a difference?

Reply:
Added the case with CH and a different profile

Reply:
Ok thanks